app/models/user.rb

```diff
@@ -71,4 +75,37 @@ def user_setting
   def confirmation_required?
     false
   end
+
+  private
+
+  def connect_to_mailchimp
+    gb = Gibbon::API.new
+    if self.mailing_list && self.confirmed?
+      list = gb.lists.list({ filters: { list_name: 'OpenFarm Subscribers' } })
+      gb.lists.subscribe({ id: list['data'][0]['id'],
+                           email: { email: self[:email] },
+                           merge_vars: { :DNAME => self[:display_name] },
+                           double_optin: false,
+                           update_existing: true })
+
+    end
+    if self.help_list && self.confirmed?
+      list = gb.lists.list({ filters: { list_name: 'OpenFarm Helpers' } })
```

Comment:
Useless assignment to variable - `list`.<br>Redundant curly braces around a hash parameter.